server/handlers/instrumented_project_command_output_handler.go

```diff
@@ -1,44 +1,33 @@
 package handlers
 
 import (
+	"fmt"
+
 	stats "github.com/lyft/gostats"
-	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
 )
 
 type InstrumentedProjectCommandOutputHandler struct {
 	ProjectCommandOutputHandler
-	numChans stats.Gauge
+	numWSConnnections stats.Gauge
+	logger            logging.SimpleLogging
 }
 
-func NewInstrumentedProjectCommandOutputHandler(prjCmdOutputHandler ProjectCommandOutputHandler, statsScope stats.Scope) ProjectCommandOutputHandler {
+func NewInstrumentedProjectCommandOutputHandler(prjCmdOutputHandler ProjectCommandOutputHandler, statsScope stats.Scope, logger logging.SimpleLogging) ProjectCommandOutputHandler {
 	return &InstrumentedProjectCommandOutputHandler{
 		ProjectCommandOutputHandler: prjCmdOutputHandler,
-		numChans:                    statsScope.Scope("log_streaming").NewGauge("num_ws_chans"),
+		numWSConnnections:           statsScope.Scope("job").Scope("websocket").NewGauge("connections"),
```

Comment:
Shouldn't this also be under the scope `atlantis.api.getprojectjobs`?

so it would look like:
 `atlantis.api.getprojectjobs.job.websocket.connections

Reply:
Yes, I've setup the api scope in when creating the handler in server.go since the cmd scope is also being set similarly. https://github.com/lyft/atlantis/blob/821c810b810f94b99b4623113ed625019a2c01ec/server/server.go#L324-L331
https://github.com/lyft/atlantis/blob/821c810b810f94b99b4623113ed625019a2c01ec/server/server.go#L639-L656

Reply:
so you're saying the scope is now `atlantis.api.job.websocket.connections`?

can we put it under getprojectjobs?